tools/v8-prof/linux-tick-processor

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+TEMP_SCRIPT_FILE="/tmp/node-tick-processor-input-script"
+tools_path=`cd $(dirname "$0");pwd`
+v8_tools="$tools_path/../../deps/v8/tools"
+log_file="v8.log"
+for arg in "$@"
+do
+  if ! expr "X${arg}" : "^X-" > /dev/null; then
```

Comment:
It turns out the ^ was completely unnecessary as it is added implicitly by the expr command. It has been removed.